### PR TITLE
general: Fix broken links.

### PR DESCRIPTION
**Pull request by InterLinked1**
This fixes a number of broken links throughout the tree, mostly caused by wiki.asterisk.org being replaced with docs.asterisk.org, which should eliminate the need for sporadic fixes as in f28047db36a70e81fe373a3d19132c43adf3f74b.

Resolves: #430

### CODE REVIEW

**Comment by InterLinked1** [comment]
cherry-pick-to: 18
cherry-pick-to: 20
cherry-pick-to: 21

File: README.md [hunk 1]
```diff
@@ -258,7 +258,7 @@ Asterisk is a trademark of Sangoma Technologies Corporation
 
 [home page]: https://www.asterisk.org
 [support]: https://www.asterisk.org/support
-[wiki]: https://wiki.asterisk.org/
+[wiki]: https://docs.asterisk.org/
```
Comment (seanbright) [inline README.md:261]: It's not a wiki

File: configs/samples/pjsip.conf.sample [hunk 2]
```diff
@@ -1510,8 +1510,6 @@
 
 ; MODULE PROVIDING BELOW SECTION(S): res_pjsip_pubsub
 ;=============================RESOURCE-LIST===================================
-; See https://wiki.asterisk.org/wiki/pages/viewpage.action?pageId=30278158
-; for more information.
```
Comment (seanbright) [inline configs/samples/pjsip.conf.sample:1514]: Don't delete without a replacement

File: configs/samples/res_snmp.conf.sample [hunk 3]
```diff
@@ -9,10 +9,6 @@
 ;
 ; If you use the subagent model, you need to enable agentx in snmpd.conf
 ; Note that you can only run one Asterisk on the system in this case.
-;
-; See https://wiki.asterisk.org/wiki/display/AST/Simple+Network+Management+Protocol+(SNMP)+Support
-; to get more information about
-; snmp support in Asterisk
```
Comment (seanbright) [inline configs/samples/res_snmp.conf.sample:15]: Don't delete

File: configs/samples/sla.conf.sample [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ;
 ; Configuration for Shared Line Appearances (SLA).
 ;
-; See http://wiki.asterisk.org or doc/AST.pdf for more information.
+; See https://docs.asterisk.org or doc/AST.pdf for more information.
```
Comment (seanbright) [inline configs/samples/sla.conf.sample:4]: Pretty sure we don't generate `AST.pdf` anymore so you can remove the mention of it

File: doc/README.txt [hunk 5]
```diff
@@ -1,7 +1,7 @@
 The vast majority of the Asterisk project documentation has been moved to the
 project wiki:
```
Comment (seanbright) [inline doc/README.txt:2]: Not a wiki

File: doc/asterisk.8 [hunk 6]
```diff
@@ -248,7 +248,7 @@ https://www.asterisk.org - The Asterisk Home Page
 .PP
 http://www.asteriskdocs.org - The Asterisk Documentation Project
 .PP
-https://wiki.asterisk.org - The Asterisk Wiki
+https://docs.asterisk.org - The Asterisk Wiki
```
Comment (seanbright) [inline doc/asterisk.8:251]: Not a wiki

File: doc/asterisk.sgml [hunk 7]
```diff
@@ -427,7 +427,7 @@
    http://www.asteriskdocs.org - The Asterisk Documentation Project
   </para>
   <para>
-   https://wiki.asterisk.org - The Asterisk Wiki
+   https://docs.asterisk.org/ - The Asterisk Wiki
```
Comment (seanbright) [inline doc/asterisk.sgml:430]: Not a wiki

File: main/asterisk.c [hunk 8]
```diff
@@ -70,8 +70,8 @@
 /*!
  * \page asterisk_community_resources Asterisk Community Resources
  * \par Websites
- * \li http://www.asterisk.org Asterisk Homepage
- * \li http://wiki.asterisk.org Asterisk Wiki
+ * \li https://www.asterisk.org Asterisk Homepage
+ * \li https://docs.asterisk.org Asterisk Wiki
```
Comment (seanbright) [inline main/asterisk.c:74]: Not a wiki

File: main/stasis.c [hunk 9]
```diff
@@ -287,7 +287,7 @@
  *
  * Subscribers are sensitive to shutdown sequencing, specifically in how the
  * reference message types. This is fully detailed on the wiki at
```
Comment (seanbright) [inline main/stasis.c:289]: Not a wiki

File: res/res_pjsip/pjsip_config.xml [hunk 10]
```diff
@@ -386,7 +386,7 @@
 						</para>
 						<para>
 						A more detailed description of how this option functions can be found on
-						the Asterisk wiki https://wiki.asterisk.org/wiki/display/AST/SIP+Direct+Media+Reinvite+Glare+Avoidance
+						the Asterisk wiki https://docs.asterisk.org/Configuration/Channel-Drivers/SIP/Concepts/SIP-Direct-Media-Reinvite-Glare-Avoidance/
```
Comment (seanbright) [inline res/res_pjsip/pjsip_config.xml:389]: Not a wiki

**Review by seanbright — CHANGES_REQUESTED** [review]

File: configs/samples/pjsip.conf.sample [hunk 11]
```diff
@@ -1509,9 +1509,9 @@
 
 
 ; MODULE PROVIDING BELOW SECTION(S): res_pjsip_pubsub
-;=============================RESOURCE-LIST===================================
 ; See https://wiki.asterisk.org/wiki/pages/viewpage.action?pageId=30278158
```
Comment (jcolp) [inline configs/samples/pjsip.conf.sample:1512]: This is https://docs.asterisk.org/Configuration/Channel-Drivers/SIP/Configuring-res_pjsip/Resource-List-Subscriptions-RLS/

**Review by jcolp — CHANGES_REQUESTED** [review]

**Review by jcolp — APPROVED** [review]

**Pull request merged**